include/swift/AST/TypeCheckRequests.h

```diff
@@ -872,8 +872,8 @@ class LazyStoragePropertyRequest :
 ///
 /// Produces true if an error occurred, false otherwise.
 /// FIXME: it would be far better to return the type-checked body.
-class TypeCheckFunctionBodyUntilRequest :
-    public SimpleRequest<TypeCheckFunctionBodyUntilRequest,
+class TypeCheckFunctionBodyRequest :
+    public SimpleRequest<TypeCheckFunctionBodyRequest,
```

Comment:
Maybe we should name this `TypeCheckFunctionBodyAtLocRequest`, and pair it with `swift::typeCheckAbstractFunctionBodyAtLoc`? This will make it immediately clear that the request does not simply typecheck the whole thing.

Reply:
Agreed! done